nixos/modules/profiles/perlless.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, ... }:
+
+{
+
+  # Disable switching to a new configuration. This is not a necessary
```

Comment:
Does this mean `nixos-rebuild switch` will not work?

Reply:
Yes exactly

Reply:
Maybe we can rename the profile to `appliance` or something to stress this more?

Reply:
We already have an appliance profile. That's why I added the comment above the code. It's not a necessary limitation of the perlless profile but just a current one because switch-to-configuration has not been rewritten yet.

Reply:
What are the plans for re-writing https://switch-to-configuration.pl?

Reply:
@dasJ can tell you more about this. But AFAIU he wants to stabilize the script first (fix all the outstanding issues) and then migrate probably to Rust.